packages/typescript-generator/templates/typescript-package/vitest.config.ts.hbs

```diff
@@ -0,0 +1,8 @@
+import { defineConfig } from "vitest/config";
+
+export default defineConfig({
+  test: {
+    globals: true,
+    environment: "node",
+  },
+});
```

Comment:
IMHO enabling `globals` is a bad practice that will make the test more "implicit" and hard to understand

```suggestion
export default defineConfig({
  test: {
    environment: "node",
  },
});
```